tensorflow/core/kernels/mkl_pooling_ops_common.h

```diff
@@ -64,7 +70,9 @@ struct MklPoolingParams {
         alg_kind(alg_kind),
         prop_kind(prop_kind),
         src_format(src_format),
-        src_md(src_md) {}
+        src_md(src_md),
+        diff_dst_md(diff_dst_md) {
```

Comment:
Should this be guarded with `#ifdef ENABLE_MKLDNN_V1`?

Reply:
Addressed it.